#!/usr/bin/python3


def raise_exception_msg(message=""):
    """Raise  NameError exception with a nachricht."""
    raise NameError(message)
